bench/rsa.cc

```
@@ -0,0 +1,609 @@
+    }
+  }
+}
+
+void BM_SpeedRSAImportKeyAndVerify(benchmark::State &state,
+                                   bssl::Span<const uint8_t> der) {
+  bssl::UniquePtr<RSA> key(RSA_private_key_from_bytes(der.data(), der.size()));
+  if (!key || RSA_size(key.get()) > kMaxSignature) {
+    state.SkipWithError("Failed to parse key.");
+    return;
+  }
+  uint8_t sig[kMaxSignature];
```

Clang (14.0.6)

[warning] variable 'sig' is not initialized

(cppcoreguidelines-init-variables)